src/components/CreateUser/CreateUserForm.js

```diff
@@ -124,11 +124,46 @@ export default class CreateUserForm extends React.Component {
       .auth()
       .createUserWithEmailAndPassword(email, passHash)
       .then(({ user }) => {})
-      .catch(error => {
-        console.log(error);
-        let newErr = error.message || "";
-        newErr = newErr.replace("email address", "username");
-        this.setState({ waiting: false, errorMessage: newErr || "failed to create user" });
+      .catch(err => {
+        console.log(err);
```

Comment:
Up to you if you want to keep this log in. I thought it would be nice if teachers were able to relay this to us. Technically, we could set up something with the backend to be able to log this type of stuff but for another day.

Reply:
Agreed, I'll keep them in for now and switch it to `console.error()` so it's not just a random console log. I do wonder how we'd log this stuff eventually - definitely something to consider if we ever rewrite the backend.